pkgs/tools/misc/pkgdiff/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, perl, wdiff }:
+stdenv.mkDerivation rec{
+  version = "1.7.2";
+  name = "pkdiff-${version}";
```

```suggestion
  pname = "pkgdiff";
```
with a **g** to be consistent with upstream name